packages/react-scripts/config/eslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "react-app"
+}
```

Comment:
After ejecting we already put eslint config into the package.json (as far as I remember).
Why is this necessary?

Reply:
Great to know ... now ... after few days of working on it (madman laugh: iiihahahaha).

Reply:
This file was added to run linter before ejecting.
Do you know how is it possible to have same effect but without such a file?

*Side note: ESlinter can be run programmicaly:*
```js
const CLIEngine = require("eslint").CLIEngine;
const cli = new CLIEngine({
    configFile: require.resolve('../config/eslint.json'),
    fix: process.argv.slice(2).indexOf('--fix') >= 0
});
const report = cli.executeOnFiles(["src/**/*.{js,jsx,mjs}"]);
CLIEngine.outputFixes(report);
const formatter = cli.getFormatter();

console.log(formatter(report.results));
```
*But it still needs a config file. In [supported options](https://eslint.org/docs/developer-guide/nodejs-api#cliengine) it is possible to pass rules, but not `extends`. And, sadly, running above snippet after ejecting doesn't work, because eslint it cannot find jsx-a11y definitions, even with this line added: `plugins: ['import', 'flowtype', 'jsx-a11y', 'react']`*

Reply:
Can `baseConfig` argument help?

Reply:
Adding `plugins: ['jxs-a11y']` throws `Error: Failed to load plugin jxs-a11y: Cannot find module 'eslint-plugin-jxs-a11y' (after yarn eject)

And trying these options didn't help
```
  cwd: '/tmp/demo-15',
  rulePaths: [
    path.dirname(require.resolve('eslint-plugin-jsx-a11y')) + '/rules'
  ],
```

I wonder, why having `baseConfig: { extends: "react-app" }` in `scripts/lint.js` would be better than an explicit config file in the config directory? Both of these duplicate entry in `package.json`.